lgc/patch/PatchBufferOp.cpp

```diff
@@ -174,8 +173,6 @@ void BufferOpLowering::registerVisitors(llvm_dialects::VisitorBuilder<BufferOpLo
 void BufferOpLowering::finish() {
   // If PHI nodes on descriptors weren't optimized away, assume that divergence in the original phi was due to sync
   // divergence, and the new phi should be divergent as well.
-  //
-  // TODO: DivergenceAnalysis should really be updatable/preservable (but switch to new uniform analysis first)
```

Comment:
Keep this comment, since "UniformityAnalysis should really be updatable/preservable" too.

Reply:
Done in https://github.com/GPUOpen-Drivers/llpc/pull/2388 . I created a new pull request because I could not push in this one :)